pynest/examples/hpc_benchmark.py

```diff
@@ -224,10 +224,10 @@ def build_network(logger):
     nest.overwrite_files = True
 
     nest.message(M_INFO, "build_network", "Creating excitatory population.")
-    E_neurons = nest.Create("iaf_psc_alpha", NE, params=model_params)
+    E_neurons = nest.Create("iaf_psc_alpha_hom", NE, params=model_params)
```

Comment:
should it be a user-configurable option which version of the model to select? Should the default remain the same (as the changed proposed in this PR is a numerical approximation for the purpose of runtime performance benefit)?